main.js

```diff
@@ -88,11 +78,11 @@ function selectedDay() {
 
     h2.textContent = 'Scheduled Events for ' + selectedText;
 
-      var td = document.querySelectorAll('td');
+    var td = document.querySelectorAll('td');
 
 
-  td[0].textContent = schedule[selectedText].todo.time;
-  td[1].textContent = schedule[selectedText].todo.description;
+    td[0].textContent = schedule[selectedText].todo.time;
+    td[1].textContent = schedule[selectedText].todo.description;
```

Comment:
I understand that the wireframe has table elements by default but now that we are creating the table dynamically it is ok to have an empty table body
so you can create the elements dynamically rather than modifying existing ones.